airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/destination/buffered_stream_consumer/OnCloseFunction.kt

```diff
@@ -5,20 +5,21 @@
 package io.airbyte.cdk.integrations.destination.buffered_stream_consumer
 
 import io.airbyte.cdk.integrations.destination.StreamSyncSummary
-import io.airbyte.commons.functional.CheckedBiConsumer
 import io.airbyte.protocol.models.v0.StreamDescriptor
 
 /**
  * Interface allowing destination to specify clean up logic that must be executed after all
  * record-related logic has finished.
  *
- * The map of StreamSyncSummaries MUST be non-null, but MAY be empty. Streams not present in the map
```

Comment:
This was only needed for BufferedStreamConsumer, which should no longer be relevant to DV2 destinations...